.circleci/config.yml

```diff
@@ -0,0 +1,78 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/go/src/github.com/docker/libnetwork
+  docker:
+    - image: 'circleci/golang:1.10'
```

Comment:
oh! does this one have to be updated as well?

Reply:
no this one is irrelevant for us, I will check if I can change it to something minimal, we build everything in the container and that's the version that counts

Reply:
Alright! I'd say; let's go then 💃